docs/tools/cli/service/flink.rst

```diff
@@ -255,4 +255,4 @@ An example of ``avn service flink job list`` output:
 
   ID                                STATUS
   ================================  =======
-  b63c78c70033e00afa84de9029257e31  RUNNING
```

Comment:
hey, could you please explain why was this changed?

Reply:
i have no idea why it shows up, both lines look identical to me, let me compare whitespaces really quick

Reply:
whitespaces look identical too in both branches